### PR TITLE
Feat: added Grid

### PR DESCRIPTION
**Pull request by anjali1102**
>Simplified Grid
 - grid-2: two items in a grid
- grid-3: three items

Check grid Preview 👉👉 [here ](https://rapidui-git-grid-anjali1102.vercel.app/components/Grid/grid.html)

### CODE REVIEW

**Review by saketkum11 — APPROVED** [review]
Good to go ahead

File: components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.grid {
+  background: var(--light-gray);
+  border-radius: 0.2rem;
+  padding: 1rem;
+}
```
Comment (Tarandeep-s1ngh) [inline components/Grid/grid.css:5]: don't mix documentation css with component's css

Comment (anjali1102) [reply components/Grid/grid.css:5]: that component css only

Comment (Tarandeep-s1ngh) [reply components/Grid/grid.css:5]: Isn't this the css for the container for your component?.... `background: var(--light-gray);
  border-radius: 0.2rem;
  padding: 1rem;`
  
  How are these properties affecting the grid?

**Pull request merged**